src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -36,7 +36,8 @@ const DiscoverySearch = ({
   fetchDiscoveryData = () => {},
   fetchData,
   onCancelToken = () => {},
-  hideBackIcon = false
+  hideBackIcon = false,
+  autoFocus = true
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding an `autoFocus` prop with a default value of `true` provides more flexibility to the component. Now it can be controlled whether the input field should automatically get focus or not.
```diff
- hideBackIcon = false
+ hideBackIcon = false,
+ autoFocus = true
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->